js/layout-javascript/agenda-code.js

```diff
@@ -902,51 +906,46 @@ DynamicList.prototype.prepareToRenderLoop = function(rows) {
       }
     });
 
-    loopData.push(newObject);
-  });
-
-  // Define detail view data based on user's settings
-  var detailData = [];
-
-  dynamicData.forEach(function(obj) {
-    clonedRecords.some(function(entryData) {
```

Comment:
We already have access to the clonedRecords iteratee in the loop above, no need to re-do it here and even nested in another loop. Another second shaved off.